about.html

```diff
@@ -103,23 +105,23 @@
 
     <!-- credits section starts -->
     <section id="credits" class="section py-0">
-      <div class="title has-text-centered mb-6">
+      <div class="title has-text-centered mb-7">
```

Comment:
mb-7 isn't a bulma feature. the most margin-bottom there is is mb-6.

Reply:
corrected - thank you